.github/workflows/build_and_test.yml

```diff
@@ -163,9 +164,10 @@ jobs:
     - name: Run tests
       env: ${{ fromJSON(needs.configure-jobs.outputs.envs) }}
       run: |
-        # Hive and SQL tests become flaky when running in parallel as it's too intensive.
-        if [[ "$MODULES_TO_TEST" == "hive" ]] || [[ "$MODULES_TO_TEST" == "sql" ]]; then export SERIAL_SBT_TESTS=1; fi
-        ./dev/run-tests --parallelism 2 --modules "$MODULES_TO_TEST" --included-tags "$INCLUDED_TAGS" --excluded-tags "$EXCLUDED_TAGS"
+        # Hive "other tests" test needs larger metaspace size based on experiment.
+        if [[ "$MODULES_TO_TEST" == "hive" ]] && [[ "$COMMENT_TO_TEST" == "- other tests" ]]; then export METASPACE_SIZE=2g; fi
```

Comment:
I think you can check `EXCLUDED_TAGS` contains `SlowHiveTest` instead of adding new env `COMMENT_TO_TEST`

Reply:
Oh. Got it. Let me change it in next commit.